x-pack/plugins/apm/public/components/shared/charts/timeseries_chart.tsx

```diff
@@ -78,14 +77,15 @@ export function TimeseriesChart({
   const history = useHistory();
   const { annotations } = useAnnotationsContext();
   const { setPointerEvent, chartRef } = useChartPointerEventContext();
-  const { urlParams } = useUrlParams();
   const theme = useTheme();
   const chartTheme = useChartTheme();
 
-  const { start, end } = urlParams;
+  const xValues = timeseries
+    .map(({ data }) => data.map(({ x }) => x))
+    .flatMap((_) => _);
 
-  const min = moment.utc(start).valueOf();
-  const max = moment.utc(end).valueOf();
+  const min = d3.min(xValues);
+  const max = d3.max(xValues);
```

Comment:
Might as well use native methods
```suggestion
  const min = Math.min(...xValues);
  const max = Math.max(...xValues);
```